Dockerfile

```diff
@@ -1,8 +1,10 @@
 FROM openjdk:11
 LABEL maintainer="georg.wittberger@gmail.com"
 ARG JAR_FILE
+ARG LIQUIGRAPH_CHANGELOG
```

Comment:
Please just revert these changes.

Reply:
I didn't know Liquigraph respect the variable out-of-the-box.
I reverted the changes. Thanks.